src/garage/np/exploration_strategies/base.py

```diff
@@ -1,9 +1,72 @@
-class ExplorationStrategy:
-    def get_action(self, t, observation, policy, **kwargs):
+"""Exploration Strategy API used by off-policy algorithms."""
+import abc
+
+
+# This should be an ABC inheritting from garage.Policy, but that doesn't exist
+# yet.
+class ExplorationStrategy(abc.ABC):
```

Comment:
maybe we should be more general and call this a PolicyWrapper? There are reasons to wrap a policy, other than exploration.

Reply:
So I do think a general policy interface is emerging, and once it's clear we should define a `PolicyWrapper` type, but I think it would be a little premature to do it in this change, before we have a `Policy` type in the first place.